src/components/PostComment.jsx

```diff
@@ -0,0 +1,44 @@
+import { addComment } from '../utils/api';
+import { UserContext } from '../contexts/User';
+import { useContext, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+const PostComment = ({ setComments, setIsLoading }) => {
+	const [newComment, setNewComment] = useState('');
+	const { user, isLoggedIn } = useContext(UserContext);
+	const { article_id } = useParams();
+	const [commentMessage, setCommentMessage] = useState('');
+
+	const handleSubmit = (e) => {
+		setIsLoading(true);
+		setCommentMessage('Comment sent thank you!');
+		e.preventDefault();
+		addComment(user, newComment, article_id).then(({ newComment }) => {
+			setComments((currComments) => {
+				const newComments = [...currComments, newComment];
+				return newComments;
+			});
+			setIsLoading(false);
+			setNewComment('');
+			setTimeout(() => setCommentMessage(''), 2000);
+		});
+	};
+
+	if (isLoggedIn) {
+		return (
+			<main>
+				<p>{user} Please add a comment below!</p>
+				<form onSubmit={handleSubmit}>
+					<textarea
```

Comment:
We need some validation that there is some input. I was able to post a blank comment